src/staticanalysis/bot/static_analysis_bot/clang/format.py

```diff
@@ -104,10 +104,16 @@ def get_eligible_files():
         assert patch != {}, \
             'Empty patch'
 
+        with open('/tmp/cf.diff', 'w') as f:
+            f.write(reverse_diff)
+
         # Build `ClangFormatIssue`s
         issues = []
         for filename, diff in patch.items():
-            lines = sorted(diff.get('touched', []) + diff.get('added', []))
+            lines = sorted(diff.get('touched', []) + diff.get('added', []) + diff.get('deleted', []))
+            if not lines:
+                logger.warn('No modified lines on this file', file=filename)
```

Comment:
Maybe we should assert here, as if clang-format produces a patch with no modified lines maybe there is an error

Reply:
:+1: